QgisModelBaker/libqgsprojectgen/dataobjects/layers.py

```diff
@@ -186,3 +212,27 @@ def real_id(self):
             return self.__layer.id()
         else:
             return None
+
+    def isPureLinkTable(self, project):
```

Comment:
@signedav I am not 100% sure about that. It looks if all fields (except one, the id) of a layer are referencing fields to decide if the link table has extra attributes or not. Do you think there could be more fields than the Id which are not extra fields? Maybe like composite keys or so?

Reply:
Maybe yes. Not sure. Maybe some investigations could help. Are there for example not only `t_id` but `t_ili_tid` as well? We might have a look at it together if you want.

Reply:
I have changed the implementation in a way that `t_id` and `t_ili_tid` are considered.
With the ili2db `--t_id_Name` parameter it is possible to specify a custom id column name. So i will have to look at that to.